x-pack/plugins/ml/server/models/data_recognizer/data_recognizer.ts

```diff
@@ -107,18 +114,15 @@ export class DataRecognizer {
   modulesDir = `${__dirname}/modules`;
   indexPatternName: string = '';
   indexPatternId: string | undefined = undefined;
-  savedObjectsClient: SavedObjectsClientContract;
+  /**
+   * List of job ids that require model memory estimation
```

Comment:
this comment is now incorrect, this is a list of `ModuleJob`

Reply:
https://github.com/elastic/kibana/pull/60656/commits/085960a936a2882f011597b75bc88831d504de73